src/core/trtserver.h

```diff
@@ -640,6 +640,15 @@ TRTSERVER_InferenceRequestProviderInputBatchByteSize(
     TRTSERVER_InferenceRequestProvider* request_provider, const char* name,
     uint64_t* byte_size);
 
+/// Get datatype of the specified output tensor
```

Comment:
I don't think we need to add this. You should be able to get the datatype from the response header.